src/code/entities/bank4.asm

```diff
@@ -188,71 +188,88 @@ func_004_6E1D::
     ld   hl, wEntitiesPrivateState3Table          ; $6E2F: $21 $D0 $C2
     jp   AddEntitySpeedToPos_04.updatePosition    ; $6E32: $C3 $EF $6D
 
-func_004_6E35::
+; Inputs:
+;   bc   entity index
+;
+; Outputs:
+;   d   x distance (Link's position - entity's position)
+;   e   0x01 if Link is to the left of the entity, 0x00 otherwise
+GetEntityXDistanceToLink_04::
     ld   e, $00                                   ; $6E35: $1E $00
```

Comment:
```suggestion
;   e   DIRECTION_LEFT if Link is to the left of the entity, DIRECTION_RIGHT otherwise
GetEntityXDistanceToLink_04::
    ld   e, DIRECTION_RIGHT                       ; $6E35: $1E $00
```